src/components/vuestic-components/va-radio/VaRadio.vue

```diff
@@ -108,135 +138,141 @@ export default {
     },
   },
   methods: {
-    onClick () {
+    click (e) {
```

Comment:
Naming was correct. `onClick` or `clicked` is used for internal event handling.
`click`/`focus` is used as public method.

For your solution, if you trigger focus via ref - nothing good will happen.

Reply:
Can we do `focus` + `onFocus` here? Pretty sure focus is useful for form etc.

Reply:
fixed